spyder/widgets/status.py

```diff
@@ -233,53 +234,47 @@ def __init__(self, parent, statusbar, icon=None):
         """Status bar widget for displaying the current conda environment."""
         self._interpreter = None
         super(InterpreterStatus, self).__init__(parent, statusbar, icon=icon)
-
-    def _get_interpreter_env_info(self):
-        """Get conda environment information."""
-        try:
-            out, err = subprocess.Popen(
-                [self._interpreter, '-V'],
-                stdout=subprocess.PIPE,
-                stderr=subprocess.PIPE
-            ).communicate()
-
-            if PY3:
-                out = out.decode()
-                err = err.decode()
-        except Exception:
-            out = ''
-            err = ''
-
-        return out, err
-
-    def _process_interpreter_env_info(self):
-        """Process conda environment information."""
-        out, err = self._get_interpreter_env_info()
-        out = out or err  # Anaconda base python prints to stderr
-        out = out.split('\n')[0]
-        parts = out.split()
-
-        if len(parts) >= 2:
-            out = ' '.join(parts[:2])
-
-        if is_conda_env(pyexec=self._interpreter):
-            envs_folder = os.path.sep + 'envs' + os.path.sep
-            if envs_folder in self._interpreter:
-                if os.name == 'nt':
-                    env = os.path.dirname(self._interpreter)
-                else:
-                    env = os.path.dirname(os.path.dirname(self._interpreter))
-                env = os.path.basename(env)
-            else:
-                env = 'base'
-            env = 'conda: ' + env
-        elif running_in_mac_app(self._interpreter):
-            env = 'internal'
```

Comment:
This condition was removed in this PR. Please restore it.

Reply:
I move this condition in the `get_pyenv_list` method, which is the one that is able to detect the system interpreter.